extensions/subscriptions/src/Job/SendReplyNotification.php

```diff
@@ -43,14 +44,22 @@ public function __construct(Post $post, $lastPostNumber)
 
     public function handle(NotificationSyncer $notifications)
     {
+        /**
+         * @var SettingsRepositoryInterface
+         */
+        $settings = resolve(SettingsRepositoryInterface::class);
         $post = $this->post;
         $discussion = $post->discussion;
 
-        $notify = $discussion->readers()
+        $query = $discussion->readers()
             ->where('users.id', '!=', $post->user_id)
-            ->where('discussion_user.subscription', 'follow')
-            ->where('discussion_user.last_read_post_number', $this->lastPostNumber - 1)
-            ->get();
+            ->where('discussion_user.subscription', 'follow');
+
+        if ($settings->get('flarum-subscriptions.dont_notify_unless_caught_up', true)) {
```

Comment:
isn't the fallback here unnecessary since we specify the default using the extender?

Reply:
True -- I just added it from habit!

```suggestion
        if ($settings->get('flarum-subscriptions.dont_notify_unless_caught_up')) {
```